dashboard-for-web/articles/web-dashboard-designer-mode/designing-dashboard-items/cards/conditional-formatting.md

```diff
@@ -12,41 +12,57 @@ For the Card dashboard item, you can apply conditional formatting to the card's
-
-To create a format rule, open the **Conditional Formatting** section in the dashboard item's [Options](../../ui-elements/dashboard-item-menu.md) menu or in the [data item menu](../../ui-elements/data-item-menu.md). Click "+" to add a new format rule:
-	
-![wed-dashboard-cf-add-rule](../../../../images/wed-dashboard-cf-add-rule.png)
-	
-Specify the data item/card used to calculate a condition in the **Common** section. You can also create a format rule for one data item and apply different settings to the other data item. 
+## Supported Format Rules
+
+Format rules that can be applied to different data item types are as follows:
+* numeric 
+	* **Value**
+	* **Top-Bottom**
+	* **Average**
+	* **Expression** 
+	* **Icon Ranges**
+	* **Color Ranges**
+	* **Gradient Ranges**
+* string 
+	* **Value** (with the condition type set to _Equal To_, _Not Equal To_ or _Text that Contains_)
+	* **Expression**
+* date-time 
+	* **Value**
+	* A **Date Occurring** (for dimensions with a continuous date-time group interval)
+	* **Expression**
+	* **Icon and Color Ranges**
+	* **Color Ranges**
+	* **Gradient Ranges**
+
+Refer to the following topic for more information about format condition types: [Conditional Formatting in Web Dashboard](../../appearance-customization/conditional-formatting.md).
+## Create and Edit a Format Rule   
+
+You can create and edit format rules in the **Conditional Formatting** section that is located in the following places:
+
+* The dashboard item's [Options](../../ui-elements/dashboard-item-menu.md) menu
+
+*  [data item menu](../../ui-elements/data-item-menu.md)
+
+Refer to the following topic for information on how to create and edit format rules: [Conditional Formatting in Web Dashboard](../../appearance-customization/conditional-formatting.md).
+## Card-Specific Format Condition Settings
+
+For a Card dashboard item, you can apply conditional formatting to the card's visual elements (like Title, Subtitle, different values) and change the card's background.
 
 Available settings in the **Common** section depend on the selected **Calculated by** option:
 
-* If you select the **Card** option, specify the card by whose values the format rule is calculated. The format rule applies to the [related card](providing-data.md). The **Value Type** option enables you to specify the card's value type according to which a condition is calculated. The **expression** format condition is an exception to this rule and applies to all cards.
-* The **Data Item** option allows you to select a hidden measure or a series dimension. The rule applies to all cards in the Card item.
-
-Select a format rule type from the list to open its settings.
-	
-![wdd-grid-cf-select-rule-type](../../../../images/wdd-card-cf-select-rule.png)
+* The **Card** option specifies a data item container (a card by whose values the format rule is calculated. The **Value Type** option specifies the card's measurement according to which a condition is calculated.
```

Comment:
Instead of parentheses, could we split this first sentence into two sentences? For example, is the following technically correct?

`* The **Card** option specifies a data item container. The format rule is calculated based on the card's values. The **Value Type** option specifies the card's measurement upon which a condition is calculated.`

Reply:
```suggestion
* The **Card** option specifies a data item container. The format rule is calculated based on the card's values. The **Value Type** option specifies the card's measurement upon which a condition is calculated.
```